src/main/java/com/dreamhorizon/enchantments/enchantments/Exhaust.java

```diff
@@ -32,6 +32,10 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 
+/**
```

Comment:
Are these required? I don't really care to see author on each of the files. That's what git is for.

Reply:
No, but then we'll remove them on every file.